components/core/src/clp_s/timestamp_parser/TimestampParser.hpp

```diff
@@ -94,12 +125,11 @@ class TimestampPattern {
  *
  * - \y Zero-padded year in century (69-99 -> 1969-1999, 00-68 -> 2000-2068).
  * - \Y Zero-padded year (0000-9999).
- * - \B Full month name (e.g., January).
- * - \b Abbreviated month name (e.g., Jan).
+ * - \B{January,...,December} Month name, one of twelve in the list between {}.
  * - \m Zero-padded month (01-12).
  * - \d Zero-padded day in month (01-31).
  * - \e Space-padded day in month( 1-31).
- * - \a Abbreviated day in week (e.g., Mon).
+ * - \A{Sunday,...,Saturday} Day in week, one of seven in the list between {}.
```

Comment:
```suggestion
 * - \A{Sunday,...,Saturday} Day in a week. The names inside `{}` must be provided in order starting
 *   from Sunday (indexed at 0) and include exactly seven entries (Sunday through Saturday)
 *   splitting by `,`.
```
Adding more clarification. Some people may treat Monday as the first day in a week (e.g., people from China), lol